sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/JsonSamples/DatasetJsonSamples.cs

```diff
@@ -914,5 +914,39 @@ public class DatasetJsonSamples : JsonSampleCollection<DatasetJsonSamples>
 }
 ";
 
+        [JsonSample]
+        public const string ParquetDataset = @"
+{
+  ""name"": ""ParquetDataset"",
```

Comment:
Why is it only ParquetDataset? didn't you add another type?

Reply:
Added